server/sonar-server/src/test/java/org/sonar/server/usergroups/ws/UpdateActionTest.java

```diff
@@ -186,9 +189,8 @@ public void name_too_short() throws Exception {
   }
 
   @Test
-  public void name_too_long() throws Exception {
-    GroupDto existingGroup = groupDao.insert(dbSession, new GroupDto().setName("old-name").setDescription("Old Description"));
-    dbSession.commit();
+  public void fail_if_name_is_too_long() throws Exception {
+    GroupDto existingGroup = insertGroupInDefaultOrganization(newGroupDto().setName("old-name").setDescription("Old Description"));
 
     expectedException.expect(IllegalArgumentException.class);
```

Comment:
expectMessage is missing, the IAE could be "Group 'new-name' already exists" for all we know